src/core/model_accessing_API.jl

```diff
@@ -39,5 +51,9 @@ function remove_agent_from_model!(agent::AbstractAgent, model::VecABM)
     error("Cannot remove agents in a `StandardABM` with a vector container.")
 end
 
+function remove_agent_from_model!(agent::AbstractAgent, model::StructABM)
+    error("Cannot remove agents in a `StandardABM` with a vector container.")
+end
+
```

Comment:
```suggestion
function remove_agent_from_model!(agent::AbstractAgent, model::StructABM)
    error("Cannot remove agents in a `StandardABM` with a vector container.")
end
```

same as before, just use `Union`

Reply:
I don't see anymore remove_agent_from_model! for `StructABM`, you should add `Union{VecABM, StructABM}` in the signature